pkg/controller/migmigration/pod.go

```diff
@@ -144,6 +144,98 @@ func (t *Task) haveResticPodsStarted() (bool, error) {
 	return true, nil
 }
 
+// Delete the running Velero pods.
+// Needed to stop any pending tasks after Velero CR deletion
+func (t *Task) restartVeleroPods() error {
+	// Exit early if no stale CRs were deleted
+	if !t.Owner.Status.HasCondition(StaleVeleroCRsDeleted) {
+		return nil
```

Comment:
We should be more granular here and track src and dest deletions separately, since we may only need to restart one velero. Also, we probably only need to restart velero if we deleted an `InProgress` CR. Deleting a new or pending one probably won't need restart.

Reply:
Addressed.